packages/opal-server/opal_server/statistics.py

```diff
@@ -193,6 +230,11 @@ async def _receive_other_worker_synced_state(
                 self._rpc_id_to_client_id = response.rpc_id_to_client_id
                 self._synced_after_wakeup.set()
 
+    async def _receive_other_worker_hello_message(
+        self, subscription: Subscription, hello_message: dict
+    ):
+        self._state.servers.add(hello_message["worker_id"])
```

Comment:
When a server stops sending keep alive messages, how will it be removed from the state?

Reply:
1. When a server goes up it triggers all server to send their "hello" message.
2. When a server goes up (sends wakeup message) all other servers are zeroing their servers list.
3. There is an issue here in case servers (replicas) go down and nothing comes up instead - it won't report true numbers until something restarts. I thought that's good enough for now, thinking about that again now - hmm not sure.

Reply:
It will quickly become out-of-sync between the servers